### PR TITLE
Update changes.rst for 0.2.0

### PR DESCRIPTION
**Pull request by hmgaudecker**
### What problem do you want to solve?

Update changes.rst after the fact.

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -5,18 +5,28 @@ This is a record of all past ``gettsim`` releases and what went into them in rev
...
+------------------
 
 This will be the initial release of ``gettsim``.
 
-- Migration of the parameter database from xls to yaml
+- Set up as a package installable via conda (`@tobiasraabe
+  <https://github.com/tobiasraabe>`_) 
+- Migration of the parameter database from xls to yaml (`@mjbloemer <https://github.com/mjbloemer>`_, `@MaxBlesch
+  <https://github.com/MaxBlesch>`_)
+- Migration of test parameters from xls to csv (via some time as ods...) (`@MaxBlesch
```
[the remark this answers is not in the document]
Comment (hmgaudecker) [reply CHANGES.rst]: Makes perfect sense. Should we use clear names instead of the Github names?

**Comment by tobiasraabe** [comment]
This extension is neat to create short links to appropriate issues and probably also prs. https://www.sphinx-doc.org/en/master/usage/extensions/extlinks.html

File: CHANGES.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (mjbloemer) [reply CHANGES.rst]: Or no names at all? I rarely see author names in change logs. There is still the git history with author names and git blame plus maybe an author file somewhere if you want to know who is involved. But I have no strong preference for one way or another.

**Comment by tobiasraabe** [comment]
We could use it not only for PRs, but for users as well. Define a new role `:ghuser:user` which accepts a github user name and links to ``https://github.com/user``.

File: CHANGES.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (hmgaudecker) [reply CHANGES.rst]: You don't in projects with a small core group of contributors. Ideally, we grow into a project where this is not the case anymore. Once that is true, I find it really useful to recognize people's contributions.

And you do see it empirically, e.g. for Numpy: https://github.com/numpy/numpy/blob/master/doc/changelog/1.17.0-changelog.rst. Maybe that is the way to go, i.e., not tie it to any particular contribution? Then we could just go through the commit log in a semi-automated fashion, too.

**Comment by hmgaudecker** [comment]
> We could use it not only for PRs, but for users as well. Define a new role `:ghuser:user` which accepts a github user name and links to `https://github.com/user`.

Sounds interesting. Can you try whether :@:user would work?

**Comment by tobiasraabe** [comment]
Unfortunately, `:@:` does not work. But now it renders nicely in documentation.

**Comment by tobiasraabe** [comment]
image

**Comment by hmgaudecker** [comment]
I like this. What do you think about this in terms of readability, @mjbloemer ?

**Pull request merged**